test-cases/tests/alerts/c18-validate-api-usage-alerts.md

```diff
@@ -38,146 +38,230 @@ tags:
+    ```
 
-9. Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
+9.  Click on `Ok, how does 3scale work?` and follow the 3Scale wizard to create an API
 
 10. Once on your API overview page, click on `Integration` on the left, then on `Configuration`
 
 11. Take note of the `example curl for testing` for `Staging-APIcast`
 
-12. Trigger API Usage alerts
+12. Verify RHOAMApiUsageOverLimit alert is present
 
-    | Alert to fire                        | `numRequests` | `recipients`          |
-    | ------------------------------------ | ------------- | --------------------- |
-    | RHOAMApiUsageLevel1ThresholdExceeded | 43            | BU and Customers      |
-    | RHOAMApiUsageLevel2ThresholdExceeded | 46            | BU and Customers      |
-    | RHOAMApiUsageLevel3ThresholdExceeded | 51            | BU, Customers and SRE |
+    It takes 30 minutes for the RHOAMApiUsageOverLimit to fire. In a following step the number of requests required to trigger this
+    alert will be reached. You will be asked to note the time at that point. Ensure the alert is present by taking the following steps:
+
+    Navigate to the Prometheus Alert Console
+
+    ```shell script
+    open "https://$(oc get route prometheus-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')/alerts"
+    ```
 
+    Verify that the alert query matches the below query:
+
+    `max_over_time((increase(ratelimit_service_rate_limit_apicast_ratelimit_generic_key_slowpath_total_hits[1m]))[30m:]) / 2 > 100`
+
+13. Trigger API Usage alerts
+
+        | Alert to fire                        | `numRequests` | `recipients`          |
+        | ------------------------------------ | ------------- | --------------------- |
+        | RHOAMApiUsageLevel1ThresholdExceeded | 43            | BU and Customers      |
+        | RHOAMApiUsageLevel2ThresholdExceeded | 46            | BU and Customers      |
+        | RHOAMApiUsageLevel3ThresholdExceeded | 51            | BU, Customers and SRE |
 
     Due to the timing of the Prometheus scrape interval, getting alerts to fire on a per 1 minute basis can be challenging. To cater for this timing issue, perform the following steps:
 
-      Prepare the API request command below, replacing `$numrequests` with the count set in the table above. Also make sure to replace `DUMMY_URL` and `DUMMY_KEY` with the values retrieved from the 3Scale console previously
+    Prepare the API request command below, replacing `$numrequests` with the count set in the table above. Also make sure to replace `DUMMY_URL` and `DUMMY_KEY` with the values retrieved from the 3Scale console previously
 
-      *NOTE:* Do not run the command yet!
+    _NOTE:_ Do not run the command yet!
 
-      ```sh
-      for i in {1..$numRequests}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>; done
-      ```
+    ```shell script
+    for i in {1..$numRequests}; do curl -i https://<DUMMY_URL>//?user_key=<DUMMY_KEY>; done
```

Comment:
Not related to a change in this PR but it seems this command needs to be updated to `& done` rather than `; done`